Test.java

```diff
@@ -2,6 +2,11 @@
 	
 	public static void main(String args[])
 	{
-		System.out.println("Hello World");
+		System.out.println(getUpdatedMessage("Hello"));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS106)